cmake/SetupCocal.cmake

```diff
@@ -0,0 +1,15 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+
+find_package(COCAL)
+
+if (COCAL_FOUND)
+  message("-- Found Cocal at ${COCAL_ROOT}")
```

Comment:
should use `STATUS`, should output actually useful information, see eg `SetupGsl.cmake`.